docs/pages/guides/apps/display-frames.mdx

```diff
@@ -101,3 +101,11 @@ pnpm dev
 ### Done! 🎉
 
 ::::
+
+## Using Next.js Image Optimization for image proxying
```

Comment:
we should also include this in https://framesjs.org/guides/apps/display-frames-in-react-native

and also include a list of security properties here to check

Reply:
React native doesn't have a built-in image proxy - what should we suggest?

Reply:
would just make a comment that it's a suggestion for privacy reasons. People can always use a next.js image proxy for this and just handle it via a next.js endpoint like /proxy?img=<src>